hexa/files/api.py

```diff
@@ -253,8 +253,10 @@ def generate_download_url(bucket_name: str, target_key: str, force_attachment=Fa
     client = get_storage_client()
     gcs_bucket = client.get_bucket(bucket_name)
     blob: Blob = gcs_bucket.get_blob(target_key)
+    filename = blob.name.split("/")[-1]
```

Comment:
I'm curious about this change, why is it needed?

Reply:
otherwise the name of the filename of the download file is my_dir-dir-file.jpg and not just file.jpg